CMSgov/easi-app
# Easi 2961/system intake final business case status

# EASI-2961
-  Implement status calculation for Final Business Case
   -  Introduce Submitted and Done status instead of Completed status  
- Introduce unit tests for possible permutations of step and Final Business Case state


## How to test this change


If you desire to test this manually, you can use the postman collection to do the following
1. Create a system Intake
2. Retrieve a system intake
3. Use SQL to update the system intakes step, final business case state to verify that the correct status is displayed.
    a. Note, state is not updated by current actions. That functionality will be added  in a future ticket. 
## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
